plugins/woocommerce-admin/client/products/layout/product-field-layout.tsx

```diff
@@ -1,31 +1,14 @@
-/**
- * Internal dependencies
- */
-import { WooProductFieldItem } from './woo-product-field-item';
-
 type ProductFieldLayoutProps = {
 	fieldName: string;
 	categoryName: string;
 };
 
+// TODO: Do we need this component for anything? Only used with attributes, and now only adds this wrapper.
```

Comment:
Do you know the answer to this? This component now only adds the div wrapper, and I don't believe the class `product-field-layout` is actually being used for anything.

Reply:
Yeah I don't think we need it anymore, you can probably remove it.
Although we do probably have to move the `<div className="product-field-layout">` over as it might break styling otherwise.